linux_os/guide/system/permissions/partitions/mount_option_boot_efi_nosuid/rule.yml

```diff
@@ -29,7 +29,7 @@ references:
     stigid@ol8: OL08-00-010572
     stigid@rhel8: RHEL-08-010572
 
-platform: machine
+platform: machine and uefi
```

Comment:
How about a comment in the definition of the `uefi` platform stating that the conditionals need to be added to the remediation?

Reply:
will do

Reply:
I think the comment should be directly in the Ansible and Bash remediation, instead of in the `rule.yml`.

And I was originally thinking of having the comment in `shared/applicability/firmware.yml`: https://github.com/ComplianceAsCode/content/blob/1183f6db88b49f4aa20ef4bacf92ede2222d27f3/shared/applicability/firmware.yml#L4

So that the places where people could act upon the `uefi` platform are "guarded".

Reply:
Got it.

I'm also thinking that other rules that are using these platforms, might have the remediation `broken` because their remediation will not be guarded by the check if platform is uefi or not.

For example:
https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/bootloader-grub2/uefi/group.yml#L8
https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/bootloader-grub2/non-uefi/group.yml#L8

Reply:
This discussion is not relevant anymore.